tests/Ticket/GH2783Test.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MongoDB\Laravel\Tests\Ticket;
+
+use MongoDB\Laravel\Eloquent\Model;
+use MongoDB\Laravel\Relations\MorphTo;
+use Illuminate\Database\Eloquent\Relations\MorphOne;
```

Comment:
According to the test results, the use statement is missing.
```suggestion
use MongoDB\Laravel\Eloquent\Model;
use MongoDB\Laravel\Relations\MorphTo;
use MongoDB\Laravel\Tests\TestCase;
use Illuminate\Database\Eloquent\Relations\MorphOne;
```